Sources/SwiftSyntax/Utils.swift

```diff
@@ -84,6 +84,11 @@ public struct SourceEdit: Equatable {
 }
 
 extension RawUnexpectedNodesSyntax {
+  /// Construct a ``RawUnexpectedNodesSyntax``with the given `elements`.
+  ///
+  /// If `isMaximumNestingLevelOverflow` is `true`, the node has the
+  /// `isMaximumNestingLevelOverflow` error bit set, indicating that the parser
+  /// overflowed its maximum nesting level and thus aborted parsing.
```

Comment:
Generally: 
Have you tought of using `- parameter` in docs instead of writing it in sentences?

Reply:
I have and generally find sentences more flexible and that sentences make it easier to explain connections between parameters. For example here, my impression is that any doc comment for the `elements` parameter will be pretty pointless. All I wanted to say about `elements` has, IMO, already been said in the first element. But maybe I’m also heavily influenced by the compiler, where almost all doc comments are written as sentences…

Reply:
I see. Didn't know if there were any standards from Apples side on how docs should be written. 😁